### PR TITLE
1.1. Adding plugin schema 🔌 🎶

### PR DESCRIPTION
**Pull request by olavea**

### CODE REVIEW

**Review by raae — CHANGES_REQUESTED** [review]
Check again [the documentation](https://github.com/raae/gatsby-source-cloudinary#plugin-options):
- Make sure only required are required
- Add default where default is documented

**Review by raae — CHANGES_REQUESTED** [review]
Recheck [the documentation](https://github.com/raae/gatsby-source-cloudinary#plugin-options):
- Make sure its the correct options
- Remove required fro non required options
- Add default where a default value is documented

File: gatsby-node.js [hunk 1]
```diff
@@ -8,12 +8,12 @@ exports.pluginOptionsSchema = ({ Joi }) => {
     cloudName: Joi.string().required(),
     apiKey: Joi.string().required(),
     apiSecret: Joi.string().required(),
-    resourceType: Joi.string().required(),
-    type: Joi.string().required(),
-    maxResults: Joi.integer().required(),
-    tags: Joi.boolean().required(),
-    prefix: Joi.string().required(),
-    context: Joi.boolean().required(),
+    resourceType: Joi.string().default(image),
```
Comment (raae) [inline gatsby-node.js]: The default needs to be string, ie `resourceType: Joi.string().default("image"),` same for type.

**Review by raae — CHANGES_REQUESTED** [review]

File: gatsby-node.js (hunk 1 above)
Comment (olavea) [reply gatsby-node.js]: 👍

**Pull request closed without merging**